src/faultlog/faultlog_policy.cpp

```diff
@@ -0,0 +1,82 @@
+#include <fmt/format.h>
+
+#include <faultlog_policy.hpp>
+#include <phosphor-logging/log.hpp>
+#include <util.hpp>
+
+namespace openpower::faultlog
+{
+using ::nlohmann::json;
+using ::phosphor::logging::level;
+using ::phosphor::logging::log;
+
+/**
+ * @class FaultLogPolicy
+ *
+ * Capture hardware isolation policy and FCO value in NAG json file
+ */
+int FaultLogPolicy::populate(sdbusplus::bus::bus& bus, nlohmann::json& nagJson)
+{
+    log<level::INFO>("FaultLogPolicy::populate()");
+    try
+    {
+        json jsonPolicyVal = json::object();
+        // FCO_VALUE
+        auto method = bus.new_method_call(
+            "xyz.openbmc_project.BIOSConfigManager",
+            "/xyz/openbmc_project/bios_config/manager",
+            "xyz.openbmc_project.BIOSConfig.Manager", "GetAttribute");
+        method.append("hb_field_core_override");
+
+        std::tuple<std::string, std::variant<int64_t, std::string>,
+                   std::variant<int64_t, std::string>>
+            attrVal;
+        auto result = bus.call(method);
+        result.read(std::get<0>(attrVal), std::get<1>(attrVal),
+                    std::get<2>(attrVal));
+
+        std::variant<int64_t, std::string> attr = std::get<1>(attrVal);
```

Comment:
Could you please rename attr to fcoValAttr or something like that. It makes it easier to read. Just a suggestion

Reply:
done